tests/test_exact_computer.py

```diff
@@ -79,6 +79,52 @@ def test_exact_computer_on_soum():
         # Assert efficiency for SV
         assert (np.sum(probabilistic_values["SV"].values) - predicted_value) ** 2 < 10e-7
 
+        index = "ELC"
+        # Core only works on normalized games
+        soum_normalized = SOUM(n, n_basis_games=n_basis_games, normalize=True)
+        exact_computer = ExactComputer(n_players=n, game_fun=soum_normalized)
+
+        predicted_value = soum_normalized(np.ones(n))[0]
+
+        egalitarian_least_core = exact_computer.compute_egalitarian_least_core(index)
+        # Assert efficiency for Egalitarian Least Core
+        assert (np.sum(egalitarian_least_core.values) - predicted_value) ** 2 < 10e-7
+
+
+def test_exact_elc_on_normalized_soum():
```

Comment:
General comment to the tests: I would like to have semantic checks of the core in a separate test file (not in the `test_exact_computer.py` file) and in the `test_exact_computer.py` an additional check that sees that it works with the exact one.